app/lib/features/chat_ng/widgets/chat_bubble.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:acter/common/themes/acter_theme.dart';
+import 'package:flutter/material.dart';
+import 'package:acter/common/extensions/options.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+
+// Chat Bubble UI
+class ChatBubble extends StatelessWidget {
+  final int? messageWidth;
+  // inner content
+  final Widget child;
+  final bool isUser;
```

Comment:
I don't understand what these mean. Could you please add inline documentation what `isUser` and `showAvatar` are meant to convey?